### ISSUE TITLE
stop action is not specific enough

### ISSUE THREAD

**Issue by jsmartin** [basho/node_package#30]
Currently the "stop" action of the init script uses the following logic:

```
RETVAL=`pidof beam.smp`
```

This wouldn't be a problem if only one basho service was running on the node, as there would be only one beam.smp.  The problem here is if stanchion, riak_cs, riak, or any combination thereof are running together on the same machine, it will never grab the correct pid of the process.  What ends up happening in that case is that the init script will always fail when the stop action is issued because it will never stop the proper service.

My proposed fix would be to be more specific about the beam.smp I want using the pgrep tool:

```
RETVAL=`pgrep -U stanchion beam.smp`
```

This forces it to get the pid of the beam.smp that belongs to the riakcs user.  Same thing could be done for stanchion and riak users.

**Comment by jaredmorrow** [comment]
Related:  basho/riak_cs#510 and basho/stanchion#47

**Comment by jaredmorrow** [comment]
Fixed in PR #39
